src/vm/CMakeLists.txt

```diff
@@ -423,7 +423,6 @@ elseif(CLR_CMAKE_TARGET_ARCH_I386)
     )
     
     set(VM_SOURCES_WKS_ARCH
-        ${ARCH_SOURCES_DIR}/jithelp.asm
```

Comment:
``jithelp.asm`` is already included in ``VM_SOURCES_WKS_ARCH_ASM`` at lin 340:
```
334 elseif(CLR_CMAKE_TARGET_ARCH_I386)
335     set(VM_SOURCES_WKS_ARCH_ASM
336         ${ARCH_SOURCES_DIR}/RedirectedHandledJITCase.asm
337         ${ARCH_SOURCES_DIR}/asmhelpers.asm
338         ${ARCH_SOURCES_DIR}/fptext.asm
339         ${ARCH_SOURCES_DIR}/gmsasm.asm
340         ${ARCH_SOURCES_DIR}/jithelp.asm
341     )
```
``VM_SOURCES_WKS_ARCH`` seems to be a list for C/C++ source files.